src/components/Social.tsx

```diff
@@ -72,9 +72,9 @@ function Social() {
             viewBox="0 0 24 24"
             fill="none"
             stroke="currentColor"
-            strokeWidth="2"
```

Comment:
React DOM names got overwritten with the HTML ones for some reason.

Reply:
I changed them manually to the `-` ones, as vscode complains. And we had a mixed style in that file (I'm also getting build errors for these)